scripts/validate.py

```diff
@@ -192,10 +164,25 @@ def get_branch_message(branch):
         return None, None
 
 
+def validate_experiment_message_id(exp_slug, branch):
+    """ This validation enforces certain naming convention for some message
+    types such as CFR in order to support the automated analysis feature of
+    Jetstream.
+    """
+    message_type, branch_message = get_branch_message(branch)
+    if branch_message is None:
+        return
+
+    if message_type == "cfr":
+        print(f"\tValidate experiment message ID for branch {branch['slug']}")
+        assert branch_message.id == f"{exp_slug}:{branch['slug']}", \
```

Comment:
Do we also have to do this for `bucket_id` is this change different from https://bugzilla.mozilla.org/show_bug.cgi?id=1649642 ?

Reply:
You're right, we do collect both for experiment users.